src/themes/units.ts

```diff
@@ -4,10 +4,12 @@ enum Units {
   Title1 = 28,
   Title2 = 22,
   Title3 = 20,
-  LineHeightMultiplier = 1.75,
   GurmukhiLatinRatio = 1.143,
+  LineHeightMultiplier = 1.25,
```

Comment:
I think we should probably split the enums into separate declarations if there are more than one definition for it^

Reply:
What do you mean?

Reply:
e.g. a `LineHeight` enum, if you deem it worthwhile:

```
export enum LineHeight {
  standard: 1.25,
  gurmukhi: 0.7
}

```

Reply:
I would rather tackle it in https://github.com/shabados/mobile/issues/224 if possible -- esp because I think we will have to go back to gurmukhi font across the board, which will make the Typography component a little easier

Reply:
Happy to do that, please add an outcome for this there